numpy/random/tests/test_random.py

```diff
@@ -255,6 +255,18 @@ def test_random_integers_max_int(self):
         desired = np.iinfo('l').max
         np.testing.assert_equal(actual, desired)
 
+    def test_random_integers_deprecated(self):
```

Comment:
Yep, one is a more invasive change than the other. Quick and dirty is the simple option, the move is somewhat more complicated, but maybe provides functionality other spots would like. The idea isn't to move these tests, but rather put `_DeprecationTestCase`, without the underscore, someplace where you can get at it. Let's wait a bit and see if we get a second opinion.